pennylane/_grad.py

```diff
@@ -135,7 +137,7 @@ def _grad_with_forward(fun, x):
         value."""
         vjp, ans = _make_vjp(fun, x)
 
-        if not vspace(ans).size == 1:
+        if vspace(ans).size != 1:
```

Comment:
Is this just a tidier way of doing things?

Reply:
yep.  my IDE was complaining.